lib/graphql/language/generation.rb

```diff
@@ -72,6 +72,12 @@ def generate(node, indent: "")
         when Nodes::VariableIdentifier
           "$#{node.name}"
         when Nodes::SchemaDefinition
+          if (!node.query || node.query == 'Query') &&
```

Comment:
we expect this to be of type `nil | String` right? Could we reflect that with 

```ruby
node.query.nil? || node.query == "Query"
```

? That would make it more clear that `false` is not an expected value here.